src/disputables/index.js

```diff
@@ -226,17 +230,16 @@ function buildDisputedActionText(app, method) {
 
 /**
  * Builds the disputed action radspec description
+ * Assumes array of transactions not empty
```

Comment:
Just curious about the motivation for this change. Can we safely assume this, and are we looking to throw an error (instead of defaulting to an empty string) if it's not the case?

Reply:
>  Can we safely assume this

Yes we can assume this since we are now early returning when the transactions array is empty 
See [here](https://github.com/aragon/court-dashboard/pull/373/files/e1a8f16cd3a826ee55d5d828e258cf8fd6b9f239..f9d01ba38e89367da5f80b9cac81c80a74693849#diff-07f7c052c786459c4930bc8491c43713R152)

> and are we looking to throw an error

Ah yeah forgot to return an error description here as well instead of the  `No description`.